### PR TITLE
change version number to use when creating scalr gem. Fix bash export…

### PR DESCRIPTION
**Pull request by andrewpatterson3001**
Superficial changes to version number and PATH export in setup instructions. Let me know if you have any questions!

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For a more permanent solution you might have in your `~/.bashrc` or
     NODE_HOME=~/tools/node/current
     ...
 
-    export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin
+export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin:$PATH
```
Comment (TheJefe) [inline bin/README.md]: The spaces at the beginning of this were meant for making this a markdown code block.

Comment (andrewpatterson3001) [inline bin/README.md]: 👍

Comment (andrewpatterson3001) [inline bin/README.md]: @TheJefe That's strange. when I preview it, it is still a code block!

Comment (TheJefe) [inline bin/README.md]: are you sure you are previewing it at your commit?
https://github.com/thinkthroughmath/scalr/blob/3e5e7e119b56a3f82066016a809192716e808a04/bin/README.md?l=100#add-to-path

**Comment by andrewpatterson3001** [comment]
@saraolives done

**Comment by andrewpatterson3001** [comment]
@TheJefe 🥇

**Pull request merged**